MeetupPlatformApi/Entities/RefreshTokenEntity.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace MeetupPlatformApi.Entities;
+
+public class RefreshTokenEntity
+{
+    public Guid Id { get; set; }
+
+    public DateTime Expires { get; set; }
+
+    public Guid UserId { get; set; }
+    
+    public UserEntity User { get; set; }
```

Comment:
Do we really need this navigation property? It's only used by the EntityTypeConfiguration.